.github/workflows/build-test.yml

```diff
@@ -113,8 +119,103 @@ jobs:
         uses: actions/download-artifact@v2
         with:
           name: vercors-debian-package
-      - run: ls
       - run: "sudo dpkg -i *.deb"
-      - run: mono --version
       - run: vercors --silicon examples/manual/fibonacci.pvl
-      - run: SPLIT=${{ matrix.testID }}/${{ env.maxTestID }} vercors --test=examples --test-workers=1 --tool carbon --tool silicon --exclude-suite=slow,medium,problem-fail,skip-travis --progress --actions-test-output
+      - run: SPLIT=${{ matrix.testID }}/${{ env.maxTestID }} vercors --test=examples --test-workers=2 --tool carbon --tool silicon --exclude-suite=slow,medium,problem-fail,skip-travis --progress --actions-test-output --enable-test-coverage --coverage-output-file=jacoco_${{ matrix.testID }}_${{ env.maxTestID }}.xml
+      - run: ls jacoco*
+      - name: Archive Java Code Coverage (jacoco_n_m.xml)
+        uses: actions/upload-artifact@v2
+        with:
+          name: vercors-jacoco-xml
+          retention-days: 1
+          path: |
+            jacoco*.xml
+
+  integration-test-windows:
+    runs-on: windows-latest
+    needs: build-test
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: "11"
+      - name: Cache LLVM and Clang
+        id: cache-llvm
+        uses: actions/cache@v2
+        with:
+          path: ${{ runner.temp }}/llvm
+          key: llvm-10.0
+      - name: Install LLVM and Clang
+        uses: KyleMayes/install-llvm-action@v1
+        with:
+          version: "10.0"
+          directory: ${{ runner.temp }}/llvm
+          cached: ${{ steps.cache-llvm.outputs.cache-hit }}
+      - name: Download VerCors binary
+        uses: actions/download-artifact@v2
+        with:
+          name: vercors-zip-package
+      - run: mkdir vercorszip
+      # --strip-components is needed because the zip contains a directory, which contains bin, examples, etc.
+      # But the name of this directory changes from time to time, because of the version number
+      # By stripping the first component of the path when extracting this directory this is not a problem
+      # --directory extracts to a given directory, ensuring the current directory is not cluttered.
+      - run: tar --strip-components=1 --directory vercorszip -xf vercors.zip
+      # Note how arguments with commas are quoted triply:
+      # - One layer gets peeled of by the .bat script
+      # - Another layer gets peeled of by powershell
+      # This prevents the commas from being interpreted as array separators, a marvelous powershell feature.
+      - run: vercorszip\bin\vercors.bat --test=examples\basic --test-workers=2 --tool carbon --tool silicon """--exclude-suite=slow,medium,problem-fail,skip-travis""" --progress --actions-test-output
```

Comment:
skip-travis is not needed anymore. 

Could you merge dev into this branch to check whether my changes are compatible with yours?